modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/view.jsp

```diff
@@ -363,4 +363,5 @@ DLViewDisplayContext dlViewDisplayContext = new DLViewDisplayContext(dlAdminDisp
 	</c:otherwise>
 </c:choose>
 
-<%@ include file="/document_library/friendly_url_changed_message.jspf" %>
\ No newline at end of file
+<%@ include file="/document_library/friendly_url_changed_message.jspf" %>
+<%@ include file="/document_library/published_document_success_message.jspf" %>
```

Comment:
Suggestion: Maybe we could use a generic jspf for all the toast messages instead of 2 files in order to simplify

Reply:
I can unify.
Any suggestion for the file name?

toast_message.jspf? 🤔

Reply:
as suggestion of @ambrinchaudhary 
"session_messages.jspf"